core/src/main/resources/org/pentaho/platform/dataaccess/datasource/wizard/public/importDialog.properties

```diff
@@ -14,7 +14,7 @@ importDialog.VALUE=Value
 importDialog.LOC_BUNDLES=Localized Bundles
 importDialog.FILE_NAME=File Name
 importDialog.IMPORT_MONDRIAN=Import Analysis
-importDialog.IMPORT_METADATA=Import Metadata
+importDialog.IMPORT_METADATA=Import Metadata/Data Source Wizard
```

Comment:
in the [demo](https://jira.pentaho.com/browse/BISERVER-14882?focusedCommentId=417200&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-417200) mentioned in the jira issue, where does the this text  **"Import Metadata/Data Source Wizard"** show ?



https://jira.pentaho.com/browse/BISERVER-14882?focusedCommentId=417200&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-417200